tests/chainer_tests/functions_tests/activation_tests/test_rrelu.py

```diff
@@ -12,24 +12,25 @@
 
 
 def _rrelu(x, creator, train):
-    r = creator.r if train else (creator.lower + creator.upper) / 2
-    return x * numpy.where(x < 0, r, 1)
+    r = creator.r if train else (creator.lower + creator.upper) / 2.0
+    xp = cuda.get_array_module(x)    
+    return x * xp.where(x < 0.0, r, 1.0)
 
 
 @testing.parameterize(*testing.product({
     'train': [True, False],
-    'shape': [(3, 2), (5, 6)],
```

Comment:
This may not be important, but is there reason that you removed (5, 6) case from this?

Reply:
Sorry, I removed (5, 6) for pass a test. But I forgot undo. I will undo.

Reply:
I undid.